server/routerlicious/packages/routerlicious-base/src/alfred/routes/api/deltas.ts

```diff
@@ -144,9 +145,19 @@ export function create(
 		),
 		verifyStorageToken(tenantManager, config, defaultTokenValidationOptions),
 		(request, response, next) => {
-			const from = stringToSequenceNumber(request.query.from);
-			const to = stringToSequenceNumber(request.query.to);
+			let from = stringToSequenceNumber(request.query.from);
```

Comment:
We no longer need to return `400`? I think return `400` now will unfortunate introduce breaking changes for a a very small amount of tenants, but it still protect us in long term so no more other tenants start to use more than 2K, what you think?

Reply:
Hey Xin, we can use another PR https://github.com/microsoft/FluidFramework/pull/16395 for the op size restriction changes and returning 400, once we make a decision (I will remove other changes from that PR). Want to keep them in 2 different PRs.
This PR is just to use default values if from or to are undefined + add caller property.

Reply:
thx for explaining and sorry for not catching what you mentioned in sync